satpy/tests/utils.py

```diff
@@ -355,3 +355,100 @@ def __call__(self, dsk, keys, **kwargs):
             raise RuntimeError("Too many dask computations were scheduled: "
                                "{}".format(self.total_computes))
         return dask.get(dsk, keys, **kwargs)
+
+
+def make_a_scene(cont_dict, daskify=False, area=True, common_attrs=None):
+    """Make a fake Scene.
+
+    Make a Scene from fake data.  Data are provided in the
+    ``cont_dict`` argument.  In ``cont_dict``, keys should be strings
```

Comment:
could this argument name have 3 more characters and be called `content_dict` ? `cont` evoked 'continuous' to me...

Reply:
Renamed to `content_dict`.